package.json

```diff
@@ -229,5 +229,8 @@
       "npx gdc-fix-copyright",
       "git add"
     ]
+  },
+  "engines": {
+    "node": "^8.10.0"
```

Comment:
Please also update the `.nvmrc` file, because now it points to a version of Node not compatible with this.

Reply:
Fixed